src/docs/implementations/datadog.adoc

```diff
@@ -63,24 +62,25 @@ management.metrics.export.datadog:
     step: 1m
 ----
 
-`DatadogConfig.hostTag()` specifies a tag key that will be mapped to https://docs.datadoghq.com/api/v1/metrics/#submit-metrics[the `host` field] when shipping metrics to Datadog.
-For example, if `DatadogConfig.hostTag()` returns "host", the tag having "host" as its key will be used.
-You can set the tag via common tags as follows:
+`DatadogConfig.hostTag()` specifies a tag key that is mapped to https://docs.datadoghq.com/api/v1/metrics/#submit-metrics[the `host` field] when shipping metrics to Datadog.
+For example, if `DatadogConfig.hostTag()` returns `host`, the tag having `host` as its key is used.
```

Comment:
I think for tag values it makes sense to use the `code` block.

Reply:
Backticks by themselves render as code, so this change does use the code font.